LocalParks.Core/Models/Shop/OrderItemModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LocalParks.Core.Models.Shop
+{
+    public class OrderItemModel
+    {
+        [Required]
+        public int ItemId { get; set; }
+        [Required]
+        public int ProductId { get; set; }
+        [Required]
+        public int Quantity { get; set; }
+        [Required]
+        public decimal UnitPrice { get; set; }
+        public int OrderId { get; set; }
+
+        public string ProductName { get; set; }
+
+        public string ImageId { get; set; }
+
+        public decimal Subtotal { get { return Math.Round(Quantity * UnitPrice, 2); } }
```

Comment:
When a developer calls a class a Model you would expect there to be no logic when calling get on a property as it is a misleading name. Mayb use a service to get the subtotal whenever needed.